examples/with-nanostores/astro.config.mjs

```diff
@@ -9,7 +9,7 @@ export default {
   },
   devOptions: {
     // port: 3000,         // The port to run the dev server on.
-    tailwindConfig: './tailwind.config.js', // Path to tailwind.config.js if used, e.g. './tailwind.config.js'
+    // tailwindConfig: './tailwind.config.js', // Path to tailwind.config.js if used, e.g. './tailwind.config.js'
```

Comment:
this was failing to build because tailwind wasn't installed. It didn't seem to be used here, so I think that this was a bug.